src/ValidationRules/AllowDenyQueryDocument.php

```diff
@@ -57,7 +67,7 @@ public function getVisitor( ValidationContext $context ) {
 						$context->reportError(
 							new Error(
 								self::deniedDocumentMessage(),
-								[ $node ]
+								$node
```

Comment:
Looking deep in the graphql-php error [constructor](https://github.com/webonyx/graphql-php/blob/master/src/Error/Error.php#L87), this single $node gets converted to [$node].  That's why I feel this is value change.